### PR TITLE
Personal Portfolio

### PR DESCRIPTION
**Pull request by afikanzoro**
- Added a contact form
- Added a social media list section
- The contact form entails a text input for name, email input and a comments section
- Also includes a submit button

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -923,3 +924,127 @@ ul {
...
+  margin-top: 36px;
+  width: 131px;
+  height: 48px;
+  color: #fff;
+  background: #ff6b00;
+  padding: 12px 12px 12px 12px;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: bold;
+  margin-left: 122px;
```
Comment (Deepakdanger) [inline styles.css:1007]: - [ ] Let remove the border default color `black` and give some border-color same as the original website. 👇 
- Original website
yuguy

- Your website
hgjhj

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @team, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Good PR✔️
- Your website looks professional✔️
- All linter is passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi Team👌, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🎉🎉
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]
Hi @afikanzoro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

sorry for any inconvenience made by the previous code review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**